### ISSUE TITLE
[APP SUBMITTED]: dbm.error: db file doesn't exist; use 'c' or 'n' flag to create a new db

### ISSUE THREAD

**Issue by akosasante** [pymedusa/Medusa#9630]
### INFO
**Python Version**: `3.8.6 (tags/v3.8.6:db45529, Sep 23 2020, 15:52:53) [MSC v.1927 64 bit (AMD64)]`
**Operating System**: `Windows-10-10.0.19041-SP0`
**Locale**: `cp1252`
**Branch**: [master](../tree/master)
**Database**: `44.18`
**Commit**: pymedusa/Medusa@4d9c7927ebeb425e386b122a230c849163d065c7
**Link to Log**: **No Log available**
### ERROR
<pre>
2021-06-13 18:04:25 ERROR    Thread_3 :: [4d9c792] Could not connect to Trakt service: db file doesn't exist; use 'c' or 'n' flag to create a new db
Traceback (most recent call last):
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/**********/show/recommendations/trakt.py#L143">**********\show\recommendations\trakt.py</a>", line 143, in fetch_popular_shows
    trending_shows.append(self._create_recommended_show(
  File "<D:\Medusa\Medusa\ext\decorator.py:decorator-gen-52>", line 2, in _create_recommended_show
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/ext/dogpile/cache/region.py#L1577">ext\dogpile\cache\region.py</a>", line 1577, in get_or_create_for_user_func
    return self.get_or_create(
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/ext/dogpile/cache/region.py#L1042">ext\dogpile\cache\region.py</a>", line 1042, in get_or_create
    with Lock(
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/ext/dogpile/lock.py#L185">ext\dogpile\lock.py</a>", line 185, in __enter__
    return self._enter()
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/ext/dogpile/lock.py#L87">ext\dogpile\lock.py</a>", line 87, in _enter
    value = value_fn()
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/ext/dogpile/cache/region.py#L977">ext\dogpile\cache\region.py</a>", line 977, in get_value
    value = self._get_from_backend(key)
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/ext/dogpile/cache/region.py#L1265">ext\dogpile\cache\region.py</a>", line 1265, in _get_from_backend
    self.backend.get_serialized(key)
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/ext/dogpile/cache/backends/file.py#L217">ext\dogpile\cache\backends\file.py</a>", line 217, in get_serialized
    with self._dbm_file(False) as dbm_obj:
  File "C:\Users\Cam\AppData\Local\Programs\Python\Python38\lib\contextlib.py", line 113, in __enter__
    return next(self.gen)
  File "D:\Medusa\Medusa\<a href="../blob/4d9c7927ebeb425e386b122a230c849163d065c7/ext/dogpile/cache/backends/file.py#L213">ext\dogpile\cache\backends\file.py</a>", line 213, in _dbm_file
    with dbm.open(self.filename, "w" if write else "r") as dbm_obj:
  File "C:\Users\Cam\AppData\Local\Programs\Python\Python38\lib\dbm\__init__.py", line 85, in open
    raise error[0]("db file doesn't exist; "
dbm.error: db file doesn't exist; use 'c' or 'n' flag to create a new db
</pre>
---
_STAFF NOTIFIED_: @pymedusa/support @pymedusa/moderators

**Comment by medariox** [comment]
Update to the new version a see if it fixes it.

**Comment by akosasante** [comment]
@medariox
I checked for updates so I _believe_ I'm on latest but I'm still seeing this error when I navigate to: `/addShows/trendingShows/?traktList=anticipated`:
<img width="926" alt="Screen Shot 2021-06-20 at 8 27 49 PM" src="https://user-images.githubusercontent.com/9094098/122693037-14785800-d206-11eb-8655-7d9c59aeb5a9.png">

**Comment by medariox** [comment]
https://github.com/pymedusa/Medusa/issues/8759
Isn't this the same issue? Did you try to restart?
We released a new version a few days ago, so you should definitely have an update available.

**Comment by akosasante** [comment]
Ah in the end, I followed these steps and was able to get things working again, thank you!

For posterity/future users:
- Turn off Medusa (previously, I hadn't realized that with [non-sucking service manager](https://nssm.cc/), I need to stop it using NSSM instead of just the Medusa menu)
- In Medusa/Data folder, I deleted the  cache folder and the cache.db file. Actually, I just renamed them `cache-backup` and `cache.db.backup` respectively
- restarted Medusa
- Now instead, on the Trakt page I get `2021-07-12 10:52:02 WARNING  Thread_0 :: [db102f3] Unable to get trakt user, error: Forbidden - invalid API key or unapproved app` error
- Go to Config menu -> Notifications -> Social
- Follow the steps there to reconnect to my trakt account
- Voila!
